internal/resource/config/idp/idp_default_urls_resource.go

```diff
@@ -40,6 +41,10 @@ type idpDefaultUrlsResourceModel struct {
 	IdpErrorMsg      types.String `tfsdk:"idp_error_msg"`
 }
 
+type idpDefaultUrlsIdModel struct {
```

Comment:
We don't need separate structs for each resource for this. We could probably move the GetAttribute logic for getting the id types.String to the common id package